www/pages/index.html.erb

```diff
@@ -1,40 +1,47 @@
-<h2>Sequel: The Database Toolkit for Ruby (v<%= Sequel::VERSION %>)</h2>
+<section class="index">
+<div class="index__hero">
 
-<h3 id='features'>Features:</h3>
+<h1 class="index__h1"><strong>Sequel:</strong> The Database Toolkit for Ruby</h1>
 
-<ul>
-<li>Sequel provides thread safety, connection pooling and a concise DSL for constructing SQL queries and table schemas.</li>
-<li>Sequel includes a comprehensive ORM layer for mapping records to Ruby objects and handling associated records.</li>
-<li>Sequel supports advanced database features such as prepared statements, bound variables, stored procedures, savepoints, two-phase commit, transaction isolation, primary/replica configurations, and database sharding.</li>
-<li>Sequel currently has adapters for ADO, Amalgalite, IBM_DB, JDBC, MySQL, Mysql2, ODBC, Oracle, PostgreSQL, SQLAnywhere, SQLite3, and TinyTDS.</li>
-</ul>
+<a href="https://github.com/jeremyevans/sequel/" class="index__a" target="_blank">v<%= Sequel::VERSION %></a>
 
-<h3 id='a_short_example'>A short example:</h3>
+<nav class="index__nav">
+<a href="documentation.html" class="index__button">Documentation</a>
+<a href="index.html#example" class="index__button index__button--secondary">Code Sample</a>
+</nav>
+</div>
 
-<pre><code class='ruby' lang='ruby'><span class='ident'>require</span> <span class='punct'>&quot;</span><span class='string'>sequel</span><span class='punct'>&quot;</span>
+<ul class="index__ul">
+<li class="index__li">Thread safety, connection pooling and a concise DSL for constructing SQL queries and table schemas.</li>
+<li class="index__li">Comprehensive ORM layer for mapping records to Ruby objects and handling associated records.</li>
+<li class="index__li">Advanced database features such as prepared statements, bound variables, stored procedures, savepoints, two-phase commit, transaction isolation, primary/replica configurations, and database sharding.</li>
+<li class="index__li">With adapters for ADO, Amalgalite, IBM_DB, JDBC, MySQL, Mysql2, ODBC, Oracle, PostgreSQL, SQLAnywhere, SQLite3, and TinyTDS.</li>
+</ul>
 
-<span class='comment'># connect to an in-memory database</span>
-<span class='constant'>DB</span> <span class='punct'>=</span> <span class='constant'>Sequel</span><span class='punct'>.</span><span class='ident'>sqlite</span>
+<a name="example"></a>
+<pre class="pre"><code class="ruby" lang="ruby"><span class="ident">require</span> <span class="punct">&quot;</span><span class="string">sequel</span><span class="punct">&quot;</span>
```

Comment:
Converted all single quotes to double quotes so it's consistent with the other HTML files.